scripts/generate_bge_exome_calling_regions.sh

```diff
@@ -107,6 +107,17 @@ java -jar $picard IntervalListTools -UNIQUE \
 
 #Produced 296381 intervals totalling 165239993 bases.
 
+#Limit intervals to chr1-22,chrX,chrY (remove alts and chrM)
```

Comment:
have we confirmed with MPG and GP folks that they don't have objections to removing chrM, since it was included in the old exome calling regions?  (I don't expect objections, but we should make sure they are aware)

Reply:
Ah, I didn't realize they were also in the old calling regions. One conclusion here could be to just leave them in. It's possible you'd still get homoplasmic alt calls (although they'll be output as diploid `1/1`). 

Alternatively, @kachulis who from the GP side do you think is the right person to reach out to about this?

@ldgauthier Is an ATGU meeting the right place to get MPG buy in that it's ok to remove chrM from the exome calling interval list or do you think documenting it through GP's process will likely be enough since this should be a non-controversial issue?

Reply:
I think the same group we talked to about the initial changes.  Maybe just follow up in that same email thread?

Reply:
The only comment from that email thread was that we should document the change so I'm comfortable removing chrM from the interval list.

@jessicaway I'll be sure to write up and a share a blurb with you for the readme that will document this change.